src/downshift.js

```diff
@@ -659,6 +680,9 @@ class Downshift extends Component {
       ...eventHandlers,
       ...rest,
       id: this.inputId,
+      [refKey]: preval`module.exports = process.env.BUILD_REACT_NATIVE === 'true'`
+        ? this.inputRef
+        : null,
```

Comment:
I'd say let's remove the condition here and just always apply the `ref` here.

But we need to make sure we call both the given `ref` as well as our own, so could you do this?

1. Add a new function to `utils` called `callAll`:

```javascript
/**
 * Function that returns a function which calls all given functions
 * with the given arguments arguments
 */
function callAll(...fns) {
  return (...args) =>
    fns.filter(Boolean).forEach(fn => {
      if (fn) {
        fn(...args)
      }
    })
}
```

2. Add that function to the imports at the top, then use it here:

```javascript
[refKey]: callAll(this.inputRef, rest[refKey])
```

3. Bonus points: do the same thing to the `[refKey]: ...` above in `getRootProps` :+1: